Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_impl.cs

```diff
@@ -127,30 +125,32 @@ public partial class PictureFlusher {
             this._currentLayer = layer;
 
             if (paint.backdrop != null) {
-                if (paint.backdrop is _BlurImageFilter) {
-                    var filter = (_BlurImageFilter) paint.backdrop;
+                var points = new uiOffset[4];
```

Comment:
Will this cause GC?

Reply:
uiOffset is struct, it should be gc free ?

Reply:
I'm not sure about an array of structs.

Reply:
has tested a bit and turns out that array of structs is not gc free. try to fix this in the newer commit and please review the changes. Thanks a lot :-)